layer-atlas/build.gradle

```diff
@@ -45,36 +45,38 @@ ext {
 
 dependencies {
     // Layer SDK
-    compile('com.layer.sdk:layer-sdk:4.0.2') {
+    implementation('com.layer.sdk:layer-sdk:4.0.2') {
         exclude group: 'com.google.firebase'
     }
-    compile 'org.slf4j:slf4j-nop:1.7.2'
+    implementation 'org.slf4j:slf4j-nop:1.7.12'
 
     // Google Location
-    compile 'com.google.android.gms:play-services-location:11.8.0'
-    compile 'com.google.firebase:firebase-messaging:11.8.0'
+    implementation 'com.google.android.gms:play-services-location:15.0.1'
+    implementation 'com.google.firebase:firebase-messaging:17.0.0'
 
     // Image cache and loading
-    compile 'com.squareup.picasso:picasso:2.5.2'
+    implementation 'com.squareup.picasso:picasso:2.71828'
 
     // Image popup
-    compile 'com.davemorrissey.labs:subsampling-scale-image-view:3.4.1'
+    implementation 'com.davemorrissey.labs:subsampling-scale-image-view:3.4.1'
 
     // Android UI
-    compile 'com.android.support:recyclerview-v7:27.1.1'
-    compile 'com.android.support:appcompat-v7:27.1.1'
-    compile 'com.android.support:design:27.1.1'
+    implementation 'com.android.support:recyclerview-v7:27.1.1'
+    implementation 'com.android.support:appcompat-v7:27.1.1'
+    implementation 'com.android.support:design:27.1.1'
+    implementation 'com.android.support:exifinterface:27.1.1'
```

Comment:
Do we need exifinterface lib in Atlas?

Reply:
I've added the exifinterface library because this was being imported but with the wrong version, I will try to exclude that and see what brakes, also will check which dependency is adding that library.

Reply:
Just checked and we will need the exifinterface since this is used by com.davemorrissey.labs:subsampling-scale-image-view which is then used to display the images as a popup by MessagePartDecode, MessagePartRegionDecoder and AtlasImagePopupActivity